okta/resource_okta_user.go

```diff
@@ -265,6 +268,25 @@ func resourceUser() *schema.Resource {
 				Optional:    true,
 				Description: "User zipcode or postal code",
 			},
+			"password": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				Sensitive:   true,
+				ValidateFunc: validation.StringLenBetween(8, 1000), // Hope no one uses password > 1000 chars
+				Description: "User Password",
+			},
+			"recovery_question": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "User Password Recovery Question",
+			},
+			"recovery_answer": &schema.Schema{
+				Type:         schema.TypeString,
+				Optional:     true,
+				Sensitive:    true,
+				ValidateFunc: validation.StringLenBetween(4, 1000),
+				Description:  "User Password Recovery Answer",
+			},
```

Comment:
Can either of these be updated? If so should we include them in the update logic"?

Reply:
Yes, they can be updated. Let me include them in the update logic. Thanks for catching!